LaTeX/LaTeX.sublime-syntax

```diff
@@ -110,66 +122,128 @@ contexts:
 
   macros:
     - include: scope:text.tex#macros
-    - match: |-
-        (?x)
-        (
-          (\\)
-          (?:(?:new|renew|provide)command\*?)
-        )
-        (?:
-          (\{)(\\[A-Za-z@]+)(\})
-          | (\\[A-Za-z@])+
-        )
-        (?:(\[)(?:[^\]]*)(\]))*
-        (\{)
+    - match: (\\)(?:new|renew|provide)command\*?(?=[^a-zA-Z@])
```

Comment:
Negative lookaheads have the charm to also enable highlighting a valid token at eof. In this special case where `*` may terminate a command I wonder if an alternation would be better suited.

```suggestion
    - match: (\\)(?:new|renew|provide)command(?:\*|(?![A-Za-z@]))
```